### PR TITLE
Major Update

### PR DESCRIPTION
**Pull request by Sunaries**
- Fixed inconsistencies
- Improvements

### CODE REVIEW

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

File: EnglishTranslation-master/ETC.tsv [hunk 1]
```diff
@@ -14106,7 +14106,7 @@ ETC_20150803_014107	DoTimeAction:Collecting:1:WEEDING:None
 ETC_20150803_014108	Property:1/PropertyAdd:1/GiveItem:UNDER67_MQ4_ITEM01:1/NPCKill/Notice:NOTICE_Dm_GetItem:Obtained bushes:3/EffectNPC:Local:F_pc_making_finish_white:1:TOP
 ETC_20150803_014109	Give a present to Scout Master secretly without being caught
 ETC_20150803_014110	Give the demons the slip by using temporary flare
-ETC_20150803_014111	Collect Red Leaves
+ETC_20150803_014111	Collect Red Leaf
```
Comment (grillo-delmal) [inline EnglishTranslation-master/ETC.tsv:14109]: "Collect a Red Leaf" ?

Comment (Sunaries) [inline EnglishTranslation-master/ETC.tsv:14109]: "Collect Red Leaf 0/5" fits better than "Collect a Red Leaf 0/5" IMO.

"Red Leaf" is also the name of the item.

Comment (grillo-delmal) [inline EnglishTranslation-master/ETC.tsv:14109]: I suppose it's ok since is the specific item name

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by Sunaries** [comment]
Good to merge.

File: EnglishTranslation-master/ETC.tsv [hunk 2]
```diff
@@ -21757,16 +21757,16 @@ ETC_20160425_021755	Able to learn attributes instantaneously
...
-ETC_20160426_021768	Applies Token benefits to all your characters in the server for 15 days. Right-click to use.{nl}[Token Benefits]{nl} - Able to learn attributes instantaneously{nl} - Reduced Market commission fee (30%{img white_right_arrow 16 16}10%){nl} - Able to list up to 9 items on the Market{nl} - +3 increased character movement{nl} - +20% EXP from hunting monsters{nl} - Increased maximum buff count{nl} - Able to use Premium Gestures
+ETC_20160426_021759	Applies Token benefit to all your characters in the server for 30 days. Right-click to use.{nl}[Token Benefits]{nl} - Able to learn attributes instantaneously{nl} - Reduced Market commission fee (30% {img white_right_arrow 16 16} 10%){nl} - Able to list up to 10 items on the Market{nl} - +3 increased character movement{nl} - +30% EXP from hunting monsters{nl} - Able to use Premium Gestures{nl} - Increased maximum buff count{nl} - Able to use Personal Trade up to 30 times{nl} - 1 extra instance entry for missions and dungeons
```
Comment (Sousou122) [inline EnglishTranslation-master/ETC.tsv:21760]: Hi Sunaries,

As a new comer to this community, not sure it's the right way to make suggestion:
For ETC_20160426_021759, will it be better to be : "Applies Token benefit**s** to ..."
Keep consistent with following lines.

Wish your have a very good day.

Sousou

**Pull request merged**